### PR TITLE
taxonomy: origin translations

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: origin translations

### CODE REVIEW

File: taxonomies/origins.txt [hunk 1]
```diff
@@ -24,16 +24,117 @@ synonyms:en:Southwest, South West
...
+synonyms:bn:উত্তরপশ্চিম, উত্তর পশ্চিম
+synonyms:bn:উত্তরপূর্ব, উত্তর পূর্ব
+synonyms:bn:দক্ষিণপশ্চিম, দক্ষিণ পশ্চিম
+synonyms:bn:দক্ষিণপূর্ব, দক্ষিণ পূর্ব
+synonyms:bn:পূর্ব কেন্দ্রীয়, পূর্ব কেন্দ্র
+synonyms:bn:পশ্চিম কেন্দ্রীয়, পশ্চিম কেন্দ্র
+synonyms:ca:Nord-oest, Nord Oest
+synonyms:ca:Nord-est, Nord Est
+synonyms:ca:Sud-oest, Sud Oest
+synonyms:ca:Sud-est, Sud Est
```
Comment (benbenben2) [inline taxonomies/origins.txt:48]: "PartA-PartB" is same as "PartA PartB", right?
```suggestion
```

File: taxonomies/origins.txt [hunk 2]
```diff
@@ -24,16 +24,117 @@ synonyms:en:Southwest, South West
...
+synonyms:es:Noroeste, Noroeste
+synonyms:es:Noreste, Noreste
+synonyms:es:Suroeste, Suroeste
+synonyms:es:Sudeste, Sudeste
+synonyms:es:Centro Este, Este Central
+synonyms:es:Centro Oeste, Oeste Central
+synonyms:fr:Nord-Ouest, Nord Ouest
+synonyms:fr:Nord-Est, Nord Est
+synonyms:fr:Sud-Ouest, Sud Ouest
+synonyms:fr:Sud-Est, Sud Est
```
Comment (benbenben2) [inline taxonomies/origins.txt:72]: same as previous suggestion
```suggestion
```

File: taxonomies/origins.txt [hunk 3]
```diff
@@ -24,16 +24,117 @@ synonyms:en:Southwest, South West
...
+synonyms:de:Nordwest, Nord West
+synonyms:de:Nordost, Nord Ost
+synonyms:de:Südwest, Süd West
+synonyms:de:Südost, Süd Ost
+synonyms:de:Ostzentral, Ost Zentral
+synonyms:de:Westzentral, West Zentral
+synonyms:es:Noroeste, Noroeste
+synonyms:es:Noreste, Noreste
+synonyms:es:Suroeste, Suroeste
+synonyms:es:Sudeste, Sudeste
```
Comment (benbenben2) [inline taxonomies/origins.txt:66]:
```suggestion
```

File: taxonomies/origins.txt [hunk 4]
```diff
@@ -24,16 +24,117 @@ synonyms:en:Southwest, South West
...
+synonyms:it:Sudovest, Sud Ovest
+synonyms:it:Sudest, Sud Est
+synonyms:it:Centro Est, Est Centrale
+synonyms:it:Centro Ovest, Ovest Centrale
+synonyms:nl:Noordwest, Noord West
+synonyms:nl:Noordoost, Noord Oost
+synonyms:nl:Zuidwest, Zuid West
+synonyms:nl:Zuidoost, Zuid Oost
+synonyms:nl:Oost Centraal, Oost Centraal
+synonyms:nl:West Centraal, West Centraal
```
Comment (benbenben2) [inline taxonomies/origins.txt:92]:
```suggestion
```

File: taxonomies/origins.txt [hunk 5]
```diff
@@ -47,63 +148,125 @@ pt:Não especificado
...
 de:Zentralamerika
+es:América Central
 fr:Amérique centrale
 hu:Közép-Amerika
-nl:Centraal Amerika
-pl:Ameryka Centralna
+it:America Centrale
+ja:中央アメリカ
+nl:Centraal-Amerika
+pl:Ameryka Środkowa
```
Comment (benbenben2) [inline taxonomies/origins.txt:225]:
```suggestion
pl:Ameryka Środkowa, Ameryka Centralna
```

File: taxonomies/origins.txt [hunk 6]
```diff
@@ -4229,23 +4402,61 @@ wikidata:en:Q788
...
+lv:Atlantijas okeāns, Atlantiskais okeāns
+pl:Ocean Atlantycki, Atlantyk
+pt:Oceano Atlântico, Atlântico
+ro:Oceanul Atlantic, Atlantic
+sk:Atlantický oceán, Atlantik
+sl:Atlantski ocean, Atlantik
+sv:Atlanten, Atlantiska oceanen
 wikidata:en:Q97
 
 <en:Atlantic Ocean
```
Comment (benbenben2) [inline taxonomies/origins.txt:4424]: mistake?

```suggestion
<en:Atlantic Ocean
en:North Atlantic Ocean
```

File: taxonomies/origins.txt [hunk 7]
```diff
@@ -4594,23 +4904,56 @@ es:Océano Índico Oriental, Este del Océano Índico
...
+ru:Антарктика и Южный Индийский океан
+tr:Antarktika ve Güney Hint Okyanusu
+uk:Антарктика і Південний Індійський океан
 xx:FAO 58
 
 en:Pacific Ocean
 bg:Тихи океан
 ca:Oceà Pacífic
-de:Pazifische Ozean
+de:Pazifischer Ozean
```
Comment (benbenben2) [inline taxonomies/origins.txt:4927]:
```suggestion
de:Pazifischer Ozean, Pazifische Ozean
```

File: taxonomies/origins.txt [hunk 8]
```diff
@@ -4594,23 +4904,56 @@ es:Océano Índico Oriental, Este del Océano Índico
...
+tr:Pasifik Okyanusu
+uk:Тихий океан
 wikidata:en:Q98
 
 <en:Pacific Ocean
+bg:Северен Тихи океан
+ca:Oceà Pacífic Nord
+de:Nordpazifik
 en:North Pacific Ocean
+es:Océano Pacífico Norte
```
Comment (benbenben2) [inline taxonomies/origins.txt:4946]:
```suggestion
en:North Pacific Ocean
bg:Северен Тихи океан
ca:Oceà Pacífic Nord
de:Nordpazifik
es:Océano Pacífico Norte
```

File: taxonomies/origins.txt [hunk 9]
```diff
@@ -4594,23 +4904,56 @@ es:Océano Índico Oriental, Este del Océano Índico
 fr:Océan Indien Est
 xx:FAO 57
 
+bg:Антарктически и Южен Индийски океан
+ca:Antàrtida i Oceà Índic Meridional
+de:Antarktis und Südlicher Indischer Ozean
 en:Antartic and Southern Indian Ocean
```
Comment (benbenben2) [inline taxonomies/origins.txt:4910]:
```suggestion
en:Antartic and Southern Indian Ocean
bg:Антарктически и Южен Индийски океан
ca:Antàrtida i Oceà Índic Meridional
de:Antarktis und Südlicher Indischer Ozean
```

**Review by benbenben2 — CHANGES_REQUESTED** [review]
couple of changes (synonyms that are same words to remove, en: to put first, etc.)

File: taxonomies/origins.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/origins.txt:48]: not sure. @stephanegigandet why did we did that ?

Comment (teolemon) [reply taxonomies/origins.txt:48]: @benbenben2 the English version already has those synonyms. So I'd say we keep them for other languages, and remove them if we learn in the future that the parser handles that correctly ?

Comment (benbenben2) [reply taxonomies/origins.txt:48]: ok, sure

**Review by benbenben2 — APPROVED** [review]
all good

**Pull request merged**